docs/eventlisteners.md

```diff
@@ -199,6 +201,66 @@ spec:
       effect: NoSchedule
 ```
 
+### Resources
+
+The `resources` field is optional.
+Resource field helps to provide Kubernetes or custom resource information.
+
+For more info refer [TEP](https://github.com/tektoncd/community/blob/master/teps/0008-support-knative-service-for-triggers-eventlistener-pod.md)
+
+Right now `resources` field is optional to support backward compatibility with original behavior of `podTemplate`, `serviceType` and `serviceAccountName`. 
+Once we deprecate `serviceAccountName`, `serviceType` and `podTemplate` from eventlistener spec then `resources` will become mandatory field and all those information will be provided as part of this.
+
+For now `resources` has support for `kubernetesResource` but later it will have a support for Custom CRD`(ex: Knative Service)` as `customResource`
+
+```yaml
+spec:
+  resources:
+    kubernetesResource:
+      serviceType: NodePort
+      spec:
+        template:
+          metadata:
+            labels:
+              key: "value"
+            annotations:
+              key: "value"
+          spec:
+            serviceAccountName: tekton-triggers-github-sa
+            nodeSelector:
+              app: test
+            tolerations:
+            - key: key
+              value: value
+              operator: Equal
+              effect: NoSchedule
+            containers:
+            - env:
+              - name: KEY
+                value: "value"
+```
+
+With the help of `kubernetesResource` user can specify [PodTemplateSpec](https://github.com/kubernetes/api/blob/master/core/v1/types.go#L3704).
+
+Right now the allowed values as part of `podSpec` are
+```text
+ServiceAccountName
+NodeSelector
+Tolerations
+```
+as part of `containers` are
+```text
+Env
+```
+
+#### Advantages
```

Comment:
This section has useful information but I think it belongs in the TEP/design doc and not necessarily in the user docs.

Reply:
Ya that make sense removed it